packages/core/src/components/AvatarGroup/AvatarGroup.tsx

```diff
@@ -0,0 +1,155 @@
+import { Children, cloneElement, CSSProperties, forwardRef } from "react";
+
+import { css } from "@emotion/css";
+
+import { HvSize, theme } from "@hitachivantara/uikit-styles";
+
+import { useDefaultProps } from "@core/hooks/useDefaultProps";
+
+import { HvBaseProps } from "@core/types/generic";
+
+import { ExtractNames } from "@core/utils/classes";
+
+import { staticClasses, useClasses } from "./AvatarGroup.styles";
+
+import { HvAvatar } from "../Avatar/Avatar";
+
+export { staticClasses as avatarGroupClasses };
+
+export type HvAvatarGroupClasses = ExtractNames<typeof useClasses>;
+
+export interface HvAvatarGroupProps extends HvBaseProps {
+  /** Inline styles to be applied to the root element. */
+  style?: CSSProperties;
```

Comment:
I believe the `style` property is already included in the `HvBaseProps` type so we shouldn’t have to list this property again.